src/algorithms/community/label-propagation.jl

```diff
@@ -125,7 +123,7 @@ function compute_vertex_label(h::Hypergraph, v::Int64, vlabels::Dict{Int64,Int64
     max = 0
     maxL = Set{Int64}()
 
-    for e in shuffle!(rng, collect(keys(hesᵥ)))
+    for e in shuffle!(rng, sort(collect(keys(hesᵥ))))
```

Comment:
why is `sort` needed here?

Reply:
I used `sort` to be sure of iterating over either vertices or hyperedges in the same order in every run of the algorithm as its result may depend on how the vertices are explored

Reply:
OK

Reply:
@aleant93 "being deterministic" can be configured in hypergraph constructor. Hence you actually you do not need `sort` here.

Have a look at those two objects below (constructing `h2` requires `using DataStructures`):
```
julia> h = Hypergraph(2,2)
2×2 Hypergraph{Bool,Nothing,Nothing,Dict{Int64,Bool}}:
 nothing  nothing
 nothing  nothing

julia> h2 = Hypergraph{Bool,Nothing,Nothing,OrderedDict{Int64,Bool}}(2,2)
2×2 Hypergraph{Bool,Nothing,Nothing,OrderedDict{Int64,Bool}}:
 nothing  nothing
 nothing  nothing
```

The hypergraph `h` is not deterministic, however the hypergraph `h2` is fully deterministic because its elemensts `h2.v2he` and `h2.he2v` are ordered dictionaries. 
This approach taken in SimpleHypergraphs gives the users full control over result reproducibility. 

Maybe it could be just better documented and perhaps we could have an additional constructur such as:
`Hypergraph(n,k;storage_type)`. However `sort` should be removed from here.